docs_src/microservices/configuration/CommonCommandLineOptions.md

```diff
@@ -10,6 +10,9 @@ Specify local configuration directory. Default is `./res`
 
 Can be overridden with [EDGEX_CONF_DIR](./CommonEnvironmentVariables.md#edgex_conf_dir) environment variable.
 
+!!! edgey "EdgeX 3.0"
+    The `EDGEX_CONF_DIR` environment variable is replaced by `EDGEX_CONFIG_DIR` in EdgeX 3.0.
```

Comment:
Note we should also rename the `--confdir` command line option to `--confgdir`
and possibly `--file` to `--configfile`

Reply:
@FelixTing please open another issue for @lenny-intel 's suggestion in edgex-go

Reply:
created a command line flag issue in edgex-go: https://github.com/edgexfoundry/edgex-go/issues/4248

Note we should create the same issue for all microservices that use go-mod-bootstrap.